src/components/FilterSearch.tsx

```diff
@@ -157,18 +175,26 @@ export function FilterSearch({
     });
   }
 
+  const handleInputFocus = useCallback((value = '') => {
+    if (value) {
```

Comment:
I believe a blank string wouldn't have any filter search options anyway so it would just add an extra API call and necessary re-renders. I didn't of other cases where we would need to execute a new API call for a blank string so I added this guard.

Reply:
In the SDK, we invoke a filter search on focus even if the value is blank. I don't know if this was intentional or not. I would double check with Product here.

Reply:
will do